config/dev/config.yaml

```diff
@@ -7,8 +7,8 @@ web-pr-label-trigger: '"Benchmark me"'
 web-pr-label-trigger-planner-v3: '"Benchmark me (V3)"'
 web-vitess-path: /tmp
 web-mode: "development"
-web-cron-schedule: "1 1 1 1 1"
-web-cron-schedule-pull-requests: "1 1 1 1 1"
-web-cron-schedule-tags: "1 1 1 1 1"
+web-cron-schedule: "none"
+web-cron-schedule-pull-requests: "none"
+web-cron-schedule-tags: "none"
```

Comment:
That's a good point. Let's keep this like that voluntarily so we don't run the benchmarks by mistake on dev mode.